cmd/checkpoint/main.go

```diff
@@ -0,0 +1,268 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
```

Comment:
Because you're importing kubernetes tools, which all use glog -- will these work well together? e.g. errors from k8s packages are surfaced properly?